db/migrate/20170115140217_update_ems_in_miq_alert_status.rb

```diff
@@ -0,0 +1,18 @@
+class UpdateEmsInMiqAlertStatus < ActiveRecord::Migration[5.0]
+  class MiqAlertStatus < ActiveRecord::Base
+    belongs_to :resource, :polymorphic => true
```

Comment:
Technically you aren't using this relation in the migration (which is good as it can lead to dynamic loading of classes), so it shouldn't be here.  In the test you can set resource_type resource_id directory, which also avoids using that relation.